### DISCUSSION TITLE
What is the hot key to move my cursor from 1 split tab to the other?

### DISCUSSION THREAD

**Discussion by lawweiliang** [Eugeny/tabby#8148] [Q&A] [answered]
Hi community, 

Is there any keyboard's hotkey that move the cursor from one split tab to the other `without using mouse`?

![image](https://user-images.githubusercontent.com/34936929/228093328-9272045c-650e-4845-affa-f2bd70eb6f42.png)

**Comment by Eugeny** [comment, marked as answer]
Search for "focus" in Settings -> Hotkeys, there are the Mac bindings:
<img width="495" alt="image" src="https://user-images.githubusercontent.com/161476/228094080-b3d00c15-6d01-4769-8c27-4d0cd20b9eb5.png">

**Reply by lawweiliang** [reply]
Thanks @Eugeny . Problem solved. 

Love tabby and wonder how fast tabby would be in the future after migrating to Tauri. 😄
